package/etc/pylib/parser_vps_cache.py

```diff
@@ -52,7 +86,7 @@ class vpsc_dest(LogDestination):
     def init(self, options):
         self.logger = syslogng.Logger()
         try:
-            self.db = SqliteDict(f"{hostdict}.sqlite", autocommit=True)
+            self.db = SqliteDict(f"{hostdict}.sqlite", autocommit=True, decode=restricted_decode, decode_key=restricted_decode_key)
```

Comment:
It seems too many similar cases, probably have a sense to write a function that will create sqltidict and you can pass *args, **kwargs. Or make a class that will be inherit from SqliteDict and override __init__

Reply:
done